### ISSUE TITLE
SecureGate not triggering lockedBuilder when authenticated = false

### ISSUE THREAD

**Issue by Dhciolfi** [neckaros/secure_application#1]
Hi,

I am facing an issue in the following scenario:
1) I have a **SecureApplication** with the following controller:

```
final SecureApplicationController _controller = SecureApplicationController(
    SecureApplicationState(locked: true, secured: true, authenticated: false,)
);
```
2) When I open my app, the **lockedBuilder** function is not called, making my screen blur, but without the lock widget.

**Comment by neckaros** [comment]
i will look into it asap

**Comment by neckaros** [comment]
Fixed in the newly published v3.3.1

tell me if it is ok for you know

And thanks @Dhciolfi  it is a very good use case that i did not think of!
